### PR TITLE
Build fixes

### PR DESCRIPTION
**Pull request by JJL772**
* added move construction/assignment to `PyUniqueObj` to fix SWIG compiles. This requires C++11 though, so I left it behind an ifdef
* fix several tiny warnings

### CODE REVIEW

File: src/cpsw_srp_addr.cc
Comment (ernestow) [inline src/cpsw_srp_addr.cc:1]: Was the timeout to big?
0 to 18,446,744,073,709,551,615

File: src/cpsw_python.h
Comment (ernestow) [inline src/cpsw_python.h:1]: C++11 or better required

**Review by ernestow — APPROVED** [review]

File: src/cpsw_srp_addr.cc
Comment (JJL772) [reply src/cpsw_srp_addr.cc:1]: It was just a formatting specifier mismatch. `getUs` returns `unsigned long`

**Comment by JJL772** [comment]
Thanks for the review!

**Comment by ernestow** [comment]
Sounds good


On Jul 5, 2024, at 14:45, Jeremy L. ***@***.***> wrote:

﻿

Thanks for the review!

**Pull request merged**